docroot/modules/humsci/hs_megamenu/config/install/hs_megamenu.settings.yml

```diff
@@ -0,0 +1,2 @@
+hs_megamenu:
+  use_hs_megamenu: 0
```

Comment:
If this is needed, I'd rather move it into a config pages like on `/admin/config/site-options`. You can simply add a field and then any config ignore or config readonly won't need to adjusted to accommodate.

Reply:
@pookmish - for the setting, I wasn't aware you were using config_pages module, so we could certainly move that setting in there if that is best practice for Stanford. Let me know if this a blocker for release.

As for the menu item extras issue, I was told that it was added as a preemptive measure ahead of the work to be done on a separate task for adding extra things to the new megamenu. So, if we can, let's disable that module for now and when work starts on that piece, we can definitely test adding items and making sure the menu doesn't break.